tests/dao_tests/test_bigquery_sync_dao.py

```diff
@@ -429,3 +436,40 @@ def test_no_on_ehr_overrides_yes_on_dv(self):
         self._submit_dvehrconsent(self.participant_id, response_time=datetime(2019, 4, 1))
         ps_json = gen.make_bqrecord(self.participant_id)
         self.assertEqual('PARTICIPANT', ps_json['enrollment_status'])
+
+    def test_ehr_consent_expired_and_renewed(self):
+        self._set_up_participant_data(fake_time=self.TIME_1)
+        # send ehr consent expired response
+        self._submit_ehrconsent_expired(self.participant_id, response_time=self.TIME_2)
+        # send a new ehr consent (renewal/reconsent)
+        self._submit_ehrconsent(self.participant_id,
+                                response_code=CONSENT_PERMISSION_YES_CODE,
+                                response_time=self.TIME_3)
+        gen = BQParticipantSummaryGenerator()
+        ps_json = gen.make_bqrecord(self.participant_id)
+
+        self.assertIsNotNone(ps_json)
+        ehr_consents = self.get_generated_items(ps_json['consents'], item_key='consent_module',
+                                                item_value='EHRConsentPII', sort_key='consent_module_authored')
+
+        # Confirm a total of three EHR Consent responses
+        self.assertEqual(len(ehr_consents), 3)
+        # Verify the initial EHR consent details (sent by _set_up_participant_data)
+        self.assertEqual(ehr_consents[0].get('consent_module_authored', None), self.TIME_1)
+        self.assertEqual(ehr_consents[0].get('consent_value', None), CONSENT_PERMISSION_YES_CODE)
+        # This field should be None for consent payloads that don't contain the expiration hidden question code
+        self.assertIsNone(ehr_consents[0].get('consent_expired', ''))
+
+        # Verify the expired consent response details (contains the hidden expiration question code / answer value)
+        self.assertEqual(ehr_consents[1].get('consent_module_authored', None), self.TIME_2)
+        self.assertEqual(ehr_consents[1].get('consent_value', None), CONSENT_PERMISSION_NO_CODE)
+        self.assertEqual(ehr_consents[1].get('consent_expired', None), EHR_CONSENT_EXPIRED_YES)
+
+        # Verify the last EHR consent renewal;  'consent_expired' value should not be carried forward from last consent
+        self.assertEqual(ehr_consents[2].get('consent_module_authored', None), self.TIME_3)
+        self.assertEqual(ehr_consents[2].get('consent_value', None), CONSENT_PERMISSION_YES_CODE)
+        # This field should be None for consent payloads that don't contain the expiration hidden question code
+        self.assertIsNone(ehr_consents[2].get('consent_expired', ''))
+
+        self.assertIsNotNone(ps_json)
```

Comment:
This was leftover from adding a line where I could set a breakpoint.  Will remove